.github/workflows/create_containers.yml

```
@@ -50,4 +44,4 @@ jobs:
         with:
           context: .
           push: true
-          tags: "ghcr.io/fwupd/fwupd/fwupd-${{ matrix.distro }}${{ case(contains(matrix.distro, '-'), '', runner.arch == 'ARM64', '-arm64', '-amd64') }}${{ case(matrix.cross != null, format('-cross-{0}', matrix.cross), '') }}:latest"
+          tags: "ghcr.io/fwupd/fwupd/fwupd-${{ join(matrix.*, '-') }}:latest"
```

actionlint (1.7.12)

[error] object type "{}" cannot be filtered by object filtering `.*` since it has no object element

(expression)